providers/metrics/report/buffer.go

```diff
@@ -0,0 +1,52 @@
+package report
+
+import "github.com/erda-project/erda/providers/metrics/common"
+
+type buffer struct {
```

Comment:
The `buffer` is already provided in [erda-infra/pkg/parallel-writer](https://github.com/erda-project/erda-infra/blob/develop/pkg/parallel-writer/buffer.go), you don’t need to reimplement it again